tests/configuration.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+from django.conf import settings
+from django.test import TestCase
+from django.test.utils import override_settings
+
+from django_emarsys.test_helper import MockLoggingHandler
+from django_emarsys import util
+
+
+class ModelTestCase(TestCase):
+    @override_settings()
+    def test_configuration_present(self):
+        logger = logging.getLogger(util.__file__)
+        handler = MockLoggingHandler(level='WARNING')
+        logger.addHandler(handler)
+
+        # remove all EMARSYS_* settings
+        for symbol in dir(settings):
+            if symbol.startswith('EMARSYS_'):
+                delattr(settings, symbol)
+
+        util.validate_configuration()
```

Comment:
Decouple `validate_configuration` from the global django settings by changing its signature to accept a settings dict and passing `django.conf.settings` when using it. In test you can make up a settings dict on the fly without having to mock settings or deleting attributes on a module (which isn't something bad per se, just that creating a simple dict is simpler). That's an example of how decoupling increases testability.